main/UseCases/GameMaker.java

```diff
@@ -9,6 +9,7 @@
 
 public class GameMaker implements Serializable {
```

Comment:
Since GameMaker and everything inside it needs to be serializable, PMSystem should also implement serializable.

Reply:
oo right